helm/templates/config.yaml

```
@@ -10,17 +10,14 @@ data:
   AUTH_SERVER: {{ required "Valid .Values.auth.server entry required!" .Values.auth.server }}
   BAG_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.bagApi.url entry required!" .Values.bagApi.url }}
   BRP_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.brpApi.url entry required!" .Values.brpApi.url }}
-  CONTACTMOMENTEN_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.contactmomentenApi.url entry required!" .Values.contactmomentenApi.url }}
-  CONTACTMOMENTEN_API_CLIENTID: {{ required "Valid .Values.contactmomentenApi.clientId entry required!" .Values.contactmomentenApi.clientId }}
   CONTEXT_URL: {{ required "Valid .Values.contextUrl entry required!" .Values.contextUrl }}
   DB_HOST: {{ required "Valid .Values.db.host entry required!" .Values.db.host }}
   DB_NAME: {{ required "Valid .Values.db.name entry required!" .Values.db.name }}
   DB_USER: {{ required "Valid .Values.db.user entry required!" .Values.db.user }}
   GEMEENTE_CODE: {{ required "Valid .Values.gemeente.code entry required!" .Values.gemeente.code  | quote }}
   GEMEENTE_NAAM: {{ required "Valid .Values.gemeente.naam entry required!" .Values.gemeente.naam }}
   GEMEENTE_MAIL: {{ required "Valid .Values.gemeente.mail entry required!" .Values.gemeente.mail }}
-  KLANTEN_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.klantenApi.url entry required!" .Values.klantenApi.url }}
-  KLANTEN_API_CLIENTID: {{ required "Valid .Values.klantenApi.clientId entry required!" .Values.klantenApi.clientId }}
+  KLANTEN_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.klantinteractiesApi.url entry required!" .Values.klantinteractiesApi.url }}
```

```suggestion
  KLANTINTERACTIES_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.klantinteractiesApi.url entry required!" .Values.klantinteractiesApi.url }}
```

For consistency? Better I think because `klantinteracties` is the combination of klanten and contactmomenten so to speak.

---

Changed